src/directives/tooltip.js

```diff
@@ -0,0 +1,329 @@
+/*
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+
+/*
+Use the v-tooltip directive to add a tooltip to an element. The tooltip will be
+shown when the user hovers over the element or focuses it.
+
+There are a few ways to specify the text of a tooltip. In many cases, the text
+will be copied from some place where it already exists in the DOM.
+
```

Comment:
Can we list all available usage as a list here:

- v-tooltip.text
- v-tooltip.aria-describedby="someValue"
- v-tooltip.aria-label
- v-tooltip.sr-only
- v-tooltip.no-aria="someValue"